runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -947,8 +947,10 @@ There are many resources online to learn more about vim. Here's a bunch of them:
 - Vim Video-Tutorials by Derek Wyatt: http://derekwyatt.org/vim/tutorials/
 - *Learn Vimscript the Hard Way*: http://learnvimscriptthehardway.stevelosh.com/
 - *7 Habits of Effective Text Editing*: http://www.moolenaar.net/habits.html
+- *vim-galore*: https://github.com/mhinz/vim-galore
 
-If you prefer a book, *Practival Vim* by Drew Neil is recommended often.
+If you prefer a book, *Practival Vim* by Drew Neil is recommended often (and the forthcoming
```

Comment:
Practival => Practical (typo). Also I am not sure if it is a forthcoming *version* or *sequel*, no big deal though.